vagrant-nsidc.yaml

```diff
@@ -0,0 +1,7 @@
+project: collinsj_nsidc_search
```

Comment:
I know that it's still a work-in-progress, but just wanted to have this comment as a reminder to remove the "collinsj" part before final merging :)

Reply:
Whew, I did not mean to commit that! =:-!

Reply:
I removed that file entirely from the repo -- I had created it so I wouldn't have to specify "--project" in the vagrant commands.